### PR TITLE
Layout made with GRID

### PR DESCRIPTION
**Pull request by shernanm**
* Book detail design made with GRID.

## Screenshot
image

image

## Trello Card
* [Trello Card](https://trello.com/c/cZH8fNBJ)

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -32,21 +33,32 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
 .title-description {
   font-weight: bolder;
   @include font-style();
+  position: absolute;
+  top: 64px;
```
Comment (CarlosHWolox) [inline scss/detail.scss:24]: leave empty line

File: scss/detail.scss [hunk 2]
```diff
@@ -32,21 +33,32 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
...
+    @include font-style(16px);
+    top: 400px;
+  }
 }
 
 .book-details {
-  font-size: 16px;
+  font-size: 20px;
   font-weight: bold;
   margin-bottom: 30px;
```
Comment (CarlosHWolox) [inline scss/detail.scss:30]: leave empty line

File: scss/detail.scss [hunk 3]
```diff
@@ -32,21 +33,32 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
...
+  @media screen and (max-width: 1024px) {
+    font-size: 14px;
+  }
 }
 
 .description {
-  @include font-style();
+  @include font-style(18px);
```
Comment (CarlosHWolox) [inline scss/detail.scss]: use one media at end of file

File: scss/detail.scss [hunk 4]
```diff
@@ -56,31 +47,41 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
...
   width: 220px;
 }
 
 .container-book-title {
   align-items: center;
   display: flex;
+  grid-column: span 2;
+  justify-content: center;
   margin-bottom: 45px;
+  order: -1;
```
Comment (CarlosHWolox) [inline scss/detail.scss]: try not to use negative values

File: scss/detail.scss [hunk 5]
```diff
@@ -56,31 +47,41 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
...
+    top: 108px;
     transform: rotate(10deg);
     width: $badge-width;
     z-index: 1;
   }
 }
 
+.container-book {
+  display: grid;
+  grid-template: 100px / 250px 1fr;
```
Comment (CarlosHWolox) [inline scss/detail.scss]: this could be done using fr only to a more responsive screen

File: scss/detail.scss [hunk 6]
```diff
@@ -56,31 +47,41 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
     content: "";
     display: block;
     height: $badge-height;
-    left: 194px;
+    left: 192px;
```
Comment (CarlosHWolox) [inline scss/detail.scss:51]: why this change?

File: scss/detail.scss [hunk 7]
```diff
@@ -116,31 +145,48 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
...
-      top: 8px;
+      top: 64px;
       width: $badge-width-responsive;
     }
   }
 }
 
+@media screen and (max-width: 1024px) {
+  .container-book {
+    grid-template: 55px 380px 120px/1fr;
```
Comment (CarlosHWolox) [inline scss/detail.scss]: try to use more responsive measures like fr

File: scss/detail.scss [hunk 8]
```diff
@@ -116,31 +145,48 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
...
+  }
+}
+
 @media screen and (max-width: 1024px) {
   .book-img {
-    margin-right: 0;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (CarlosHWolox) [inline scss/detail.scss]: this could be done margin: 0 auto

**Review by CarlosHWolox — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 6 above)
Comment (shernanm) [reply scss/detail.scss:51]: Nothing specific just wanted to align it more.

File: scss/detail.scss [hunk 9]
```diff
@@ -100,47 +104,92 @@ $badge-width-responsive: calc(#{$badge-width} * 0.7);
...
+      height: 4px;
+      left: -58px;
+      position: absolute;
+      top: 400px;
       width: 295px;
     }
   }
 }
 
+@media screen and (max-width: 1024px) {
```
Comment (CarlosHWolox) [inline scss/detail.scss]: use only one media screen

**Review by CarlosHWolox — CHANGES_REQUESTED** [review]

**Review by CarlosHWolox — APPROVED** [review]
